pkgs/development/libraries/vtk/8.x.nix

```diff
@@ -5,5 +5,11 @@ import ./generic.nix {
   patchesToFetch = [{
    url = "https://gitlab.kitware.com/vtk/vtk/-/commit/257b9d7b18d5f3db3fe099dc18f230e23f7dfbab.diff";
    sha256 = "0qdahp4f4gcaznr28j06d5fyxiis774ys0p335aazf7h51zb8rzy";
-  }];
+  }
+  {
+    meta.description = "Fix compiling with gcc-10+";
```

Comment:
Congrats on your first PR!
Note that the meta.description is something that should be used to describe a package as opposed to a patch for a package. You can drop that line.